.project/scripts/new-tool-script/template/workflows/lint-tool-{{tool-name}}.yml

```diff
@@ -52,7 +52,7 @@ jobs:
 
     - name: Ensure we have latest changes
       run: |
-        git rebase origin/master
+        git rebase origin/${{ steps.extract_branch.outputs.branch }}
```

Comment:
incorrect, the line should be: ` git rebase origin$/{{ steps.extract_branch.outputs.branch }}` (perhaps my mistake)

Reply:
Fixed this and pulled from origin master to resolve merging issue.

Reply:
Your current changes look different to masters, check against master here:
https://github.com/holistic-web/holistic-toolbox/blob/master/.project/scripts/new-tool-script/template/workflows/lint-tool-%7B%7Btool-name%7D%7D.yml#L55

Reply:
Fixed in e86ae28